src/TodoList.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+function TodoList() {
+    const todoListData = [
```

Comment:
I really like your variable naming! Very descriptive. This is a great practice to always consider. Have your variables be as descriptive as possible. Keep in mind that as your code base and data input grows you're going to have a lot of "data". It's not necessary to include "data" in the name when you already have a very descriptive variable name. 

Rule of thumb: "Try not to have generic names for variables when possible"